src/p5xr/core/p5xr.js

```diff
@@ -189,9 +187,8 @@ export default class p5xr {
         var callMethod = function(f) {
           f.call(context);
         };
-        // TODO Just call a different function that does this minus matrix reset
         if (context._renderer.isP3D) {
-          context._renderer._update();
+          context._renderer._updatexr();
```

Comment:
It is no longer a method of the renderer right? It now belongs the p5xr.instance.